README.md

```diff
@@ -12,6 +12,8 @@ Telephony.config do |c|
   c.voice_pause_time = '0.5s'
   c.voice_rate = 'slow'
 
+  c.sender_id = 'some_sender_id'
```

Comment:
Q: Would it make sense for this to be part of the SMS config?

Reply:
So currently the SMS configs are an array, and are essentially per-region SMS credentials. Since this is across mutliple SMS configs I didn't think it made sense